src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -179,7 +180,8 @@ trait SharedFileSystem {
 
   private def stageWdlFile(call: Option[Call], wdlFile: WdlFile, hostInputsPath: Path): Try[WdlFile] = {
     val originalPath = Paths.get(wdlFile.value)
-    val executionPath = hostInputsPath.resolve(originalPath.getFileName.toString)
+    val uniqueID = UUID.randomUUID().toString.split("-")(0)
```

Comment:
Disregard, I forgot that the other backends are different than how jes does things w/ previous steps